packages/nitro-protocol/.env

```
@@ -1,20 +1,20 @@
-DEFAULT_GAS=6721975
-DEFAULT_GAS_PRICE=20000000000
-GANACHE_HOST=0.0.0.0
-GANACHE_PORT=8561
-CHAIN_NETWORK_ID=9001
-DEV_HTTP_SERVER_PORT=3000
+DEFAULT_GAS = '6721975'
+DEFAULT_GAS_PRICE = '20000000000'
+GANACHE_HOST = '0.0.0.0'
+GANACHE_PORT = '8561'
+CHAIN_NETWORK_ID = '9001'
+DEV_HTTP_SERVER_PORT = '3000'
 
 # These contract addresses get defined in the global jest setup
-NITRO_ADJUDICATOR_ADDRESS= 0x0000000000000000000000000000000000000000
-COUNTING_APP_ADDRESS= 0x0000000000000000000000000000000000000000
-SINGLE_ASSET_PAYMENT_ADDRESS= 0x0000000000000000000000000000000000000000
-CONSENSUS_APP_ADDRESS= 0x0000000000000000000000000000000000000000
-TRIVIAL_APP_ADDRESS= 0x0000000000000000000000000000000000000000
-TEST_FORCE_MOVE_ADDRESS= 0x0000000000000000000000000000000000000000
-TEST_NITRO_ADJUDICATOR_ADDRESS= 0x0000000000000000000000000000000000000000
-TEST_TOKEN_ADDRESS= 0x0000000000000000000000000000000000000000
-TEST_ETH_ASSET_HOLDER_ADDRESS= 0x0000000000000000000000000000000000000000
-TEST_TOKEN_ASSET_HOLDER_ADDRESS= 0x0000000000000000000000000000000000000000
-TEST_ASSET_HOLDER_ADDRESS = 0x0000000000000000000000000000000000000000
-TEST_ASSET_HOLDER2_ADDRESS= 0x0000000000000000000000000000000000000000
\ No newline at end of file
+NITRO_ADJUDICATOR_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+COUNTING_APP_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+SINGLE_ASSET_PAYMENT_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+CONSENSUS_APP_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TRIVIAL_APP_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_FORCE_MOVE_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_NITRO_ADJUDICATOR_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_TOKEN_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_ETH_ASSET_HOLDER_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_TOKEN_ASSET_HOLDER_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_ASSET_HOLDER_ADDRESS  = ' 0x0000000000000000000000000000000000000000'
+TEST_ASSET_HOLDER2_ADDRESS = ' 0x0000000000000000000000000000000000000000'
```

```suggestion
TEST_ASSET_HOLDER2_ADDRESS = '0x0000000000000000000000000000000000000000'
```

etc? Nitpicking, for sure: but this PR is about standardising the format.